### PR TITLE
ROX-14093: Scanner v4 support Node.js vuln matching

### PR DESCRIPTION
**Pull request by RTann**
## Checklist
- [x] Investigated and inspected CI test results
- ~[ ] Unit test and regression tests added~
- ~[ ] Evaluated and added CHANGELOG entry if required~
- ~[ ] Determined and documented upgrade steps~
- ~[ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

### Here I tell how I validated my change

Deploy Central Services. Once Matcher has all vuln data, scan `splunk:8.2.6` to find vulnerabilities

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

**Comment by RTann** [comment]
Recreation of https://github.com/stackrox/stackrox/pull/9445

**Review by dcaravel — APPROVED** [review]

**Review by dcaravel — APPROVED** [review]

File: scanner/matcher/matcher.go [hunk 1]
```diff
@@ -40,7 +40,11 @@ var (
 		"suse",
 		"ubuntu-matcher",
 	}
-)
+	if env.ScannerV4NodeJSSupport.BooleanSetting() {
```
Comment (jvdm) [inline scanner/matcher/matcher.go:43]: What about the exporter/updater?

Comment (jvdm) [reply scanner/matcher/matcher.go:43]: I guess the PR says "support matching." Let me know if you plan to add vulns to the vulnerability bundle at another time.

Comment (RTann) [reply scanner/matcher/matcher.go:43]: it's already enabled via `"osv"`

**Review by jvdm — APPROVED** [review]
LGTM.

**Pull request merged**